gallry.js

```diff
@@ -254,16 +257,16 @@ var gallry = (function() {
               var str = arr[i].src;
               var first = str.slice(0, str.length - 4); // get first part of string
               var last = str.slice(str.length - 4, str.length); // get .jpg, .png, .svg, .gif files --- must be 4 character file extension
-              el[i].style.backgroundImage = "url(" + first + "_2x" + last + ")"; // defining gallery slide image via arr.src prop
+              el[i].style.backgroundImage = "url('" + first + "_2x" + last + "')"; // defining gallery slide image via arr.src prop
```

Comment:
Good catch on the single quotes! 👍

Reply:
Thanks!  

I appreciate the offer, I'm not sure what I'd add to it (at least for now), but I'll certainly keep feeding back any improvements I make.  I was just looking for something simple / easy to implement, I don't know if a conversion to React or Vue would help (I'm only using the Bootstrap library currently for the website this is going on to, so it doesn't matter which library you're using).

Reply:
P.S.  Meant to say:  simple / easy to implement, and has the needed features!  There's a lot of simpler sliders out there that lack the nav features.